src/main/java/rx/internal/operators/OperatorSwitchIfEmpty.java

```diff
@@ -81,20 +83,7 @@ private void subscribeToAlternate() {
 
```

Comment:
I'd move this anonym inner class into a regular static class to avoid capturing unnecessary references. In addition, I'd first call ```ssub.set``` then call ```unsafeSubscribe```